server/pkg/api/accountHandlers.go

```diff
@@ -38,8 +43,19 @@ func createAccount(ctx apiContext, w http.ResponseWriter, r *http.Request) {
 
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "Error: %s\n", err.Error())
-		http.Error(w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
-		return
+		if err.Error() != "" {
```

Comment:
I like what you're doing here by sending he message back that the client can then write out. However, i think it might be better to do this a bit differently. Instead of creating a `Profile` object, maybe create a `ResponseObject` that can contain `statuscode`, `errorMessage` and `message` for now. And then write that out and then the client can just write out the errorMessage it recieves.

Reply:
Also, can you put the model in the `models/models.go` file